libbeat/processors/add_host_metadata/add_host_metadata.go

```diff
@@ -62,6 +64,32 @@ func newHostMetadataProcessor(cfg *common.Config) (processors.Processor, error)
 		data:   common.NewMapStrPointer(nil),
 	}
 	p.loadData()
+
+	if config.Geo != nil {
+		if len(config.Geo.Location) > 0 {
+			if m, _ := regexp.MatchString("^\\-?\\d+(\\.\\d+)?\\s*\\,\\s*\\-?\\d+(\\.\\d+)?$", config.Geo.Location); !m {
```

Comment:
I'm undecided if I like this one or not. It's good that we check the user input but it looks like really complex regexp where I must confess I could not easily review to make sure it's correct.

Reply:
I can break the regexp up into component strings to make ti more readable.

Reply:
FYI, the regexp isn't 100% accurate (it doesn't check the numeric bounds), but it's a lot better than nothing

Reply:
In b4feb7e I've made the regex more clear